test/conformance/gateway_conformance_test.go

```diff
@@ -50,5 +55,28 @@ func TestGatewayConformance(t *testing.T) {
 		BaseManifests:    conformanceTestsBaseManifests,
 	})
 	cSuite.Setup(t)
-	// TODO: cSuite.Run(t, tests.ConformanceTests)
+
+	t.Log("configuring gateway conformance tests")
+	tts := make([]suite.ConformanceTest, 0, len(tests.ConformanceTests))
+	for _, tt := range tests.ConformanceTests {
+		ttcpy := tt
+		for i, manifest := range ttcpy.Manifests {
+			ttcpy.Manifests[i] = fmt.Sprintf("%s/conformance/%s", manifestRepo, manifest)
```

Comment:
This seems to be overwriting the original slice - `ttcpy` contains a shallow copy of the original slice. Is this intended?

If this is intended, then I cannot see the reason for `ttcpy` existing at all - the code should yield an equivalent result if it were appending `tt` in line 66.

Suggestion: either make a deep copy of `ttcpy.Manifests` or drop `ttcpy`

Reply:
:+1: now changed in 5fea81da3013ee4458e2084fc5593ab484a836da